WalletWasabi.Tests/UnitTests/BlockchainAnalysis/CoinJoinAnonscoreTests.cs

```diff
@@ -239,4 +240,57 @@ public void InputMergePunishmentNoInheritance()
 		// 10 participants, 1 is you, your anonset would be 10 normally and now too:
 		Assert.Equal(10, tx.WalletOutputs.First().HdPubKey.AnonymitySet);
 	}
+
+	public void SiblingCoinjoinDoesntContributeToAnonScore()
+	{
+		var tx1 = new AnalyzedTransaction();
+		tx1.AddForeignInput();
+		tx1.AddWalletInput();
+		var A = tx1.AddForeignOutput();
+		var B = tx1.AddWalletOutput();
+
+		var tx2 = new AnalyzedTransaction();
+		tx2.AddForeignInput(A);
+		tx2.AddWalletInput(B);
+		tx2.AddForeignOutput();
+		var F = tx2.AddWalletOutput();
+
+		tx1.AnalyzeRecursively();
```

Comment:
This unit test is failing. If you make this change, then it'll pass, but not sure if it's correct I don't fully understand what's going on here just yet:

```suggestion
		tx2.AnalyzeRecursively();
```

Reply:
It should be `tx2`.